org.jacoco.core/src/org/jacoco/core/analysis/ILine.java

```diff
@@ -45,4 +45,10 @@
 	 */
 	int getStatus();
 
+	/**
+	 * Returns the execution count for this line
+	 *
+	 * @return execution count
+	 */
+	int getExecutionCount();
```

Comment:
I think this definition does not reflect the actual implementation. See my other comment below.